### PR TITLE
ssmsh: init at 1.4.3

### PR DESCRIPTION
**Pull request by dbirks**
###### Motivation for this change

Adding a tool that I started using recently. It allows you to browse and edit AWS Parameter Store, kind of in the same way as you would move around in a filesystem (with `cd`, `ls`, `mv`, `rm`). I've found it quite handy.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/admin/ssmsh/default.nix [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoModule, fetchurl }:
+
+buildGoModule rec {
+  pname = "ssmsh";
+  version = "1.4.3";
+
+  src = fetchurl {
```
Comment (SuperSandro2000) [inline pkgs/tools/admin/ssmsh/default.nix:7]: fetchFromGitHub can be used here right?

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 103052` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ssmsh</li>
  </ul>
</details>

File: pkgs/tools/admin/ssmsh/default.nix (hunk 1 above)
Comment (dbirks) [reply pkgs/tools/admin/ssmsh/default.nix:7]: I liked the simplicity of fetchurl, but yes you're right, it looks like most of the Go packages use fetchFromGithub, so I'll switch over to that...

Comment (dbirks) [reply pkgs/tools/admin/ssmsh/default.nix]: Ok switched over :+1:

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 103052` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>ssmsh</li>
  </ul>
</details>

**Comment by dbirks** [comment]
Thanks for testing it on both Linux and MacOS :+1:

**Pull request merged**